src/patternfly/utilities/Sizing/examples/Sizing.md

```diff
@@ -142,6 +146,70 @@ import './Sizing.css'
 | `.pf-u-h-75vh{-on-[breakpoint]}` | `*` | Sets height: 75vh |
 | `.pf-u-h-100vh{-on-[breakpoint]}` | `*` | Sets height: 100vh |
 
-## Documentation
+### Min width
+```hbs
+{{#> sizing-min-max sizing-min-max--modifier="pf-u-min-width" sizing-min-max--attribute='style="--pf-u-min-width--MinWidth: 50ch;"'}}
+    Min-width 50ch example
+{{/sizing-min-max}}
+```
+
+### Max width
+```hbs
+{{#> sizing-min-max sizing-min-max--modifier="pf-u-max-width" sizing-min-max--attribute='style="--pf-u-max-width--MaxWidth: 50ch;"'}}
+  Max-width 50ch example
+{{/sizing-min-max}}
+```
+
+### Min and max width
+```hbs
+{{#> sizing-min-max sizing-min-max--modifier="pf-u-min-width pf-u-max-width" sizing-min-max--attribute='style="--pf-u-min-width--MinWidth: 30ch; --pf-u-max-width--MaxWidth: 50ch;"'}}
+  Min-width 30ch, max-width 50ch example
+{{/sizing-min-max}}
+```
+
+### Responsive width control
+```hbs
+{{#> sizing-min-max sizing-min-max--modifier="pf-u-min-width pf-u-max-width" sizing-min-max--attribute='style="--pf-u-min-width--MinWidth: 20ch; --pf-u-max-width--MaxWidth: 30ch; --pf-u-max-width--MaxWidth-on-md: 50ch; --pf-u-max-width--MaxWidth-on-xl: 70ch;"'}}
+  Min-width 20ch, max-width 30ch, max-width-on-md 50ch, max-width-on-xl 70ch example
+{{/sizing-min-max}}
+```
+
+### Usage
+| Class                                                       | Applied to            | Outcome                       |
+| ----------------------------------------------------------- | --------------------- | ----------------------------- |
```

Comment:
Should we relabel "Class" to something like "Custom property"? (same comment applies to line 212)

Reply:
Nice